fb-install-foreman.bats

```diff
@@ -42,13 +38,8 @@ setup() {
 
 @test "stop puppet agent (if installed)" {
   tPackageExists "puppet" || skip "Puppet package not installed"
-  if tIsRHEL 6; then
-    service puppet stop; chkconfig puppet off
-  elif tIsFedora; then
-    service puppetagent stop; chkconfig puppetagent off
```

Comment:
Yes, I checked that and recent Fedora versions ship puppetagent as a symlink to puppet.